### DISCUSSION TITLE
Data processing problems with all-ions and multiple collision energies

### DISCUSSION THREAD

**Discussion by MMolinaCalle** [systemsomicslab/MsdialWorkbench#283] [Q&A]
Hi all,
I am trying to process non-dependent data adquired with multiple collision energies, however, I am encountering the following issues:
- MSDIAL v5.1.2: I select AIF adquisition for all the files, however the program crashes at the end of the aligment process and does not save any .mdproject.
- MSDIAL v5.1.0: In MS method type, I select the option "all-ions with multiple CE", however this version grashes too at the end of the aligment process.
- MSDIAL v4.9.2: This version of the program correctly processes the data, but when I want to export the alignment results only one MS/MS spectra appears. Is it only one of the CE or is it an amalgamation of all of them? and, if it is one of them, which of them? Is there any option to export all the MS/MS spectra for each compound?

I would like to find a solution to these problems with v5.1 because this version allow to load multiple libraries.

Does anyone have these same problems?

Thanks!

**Comment by pierpaolobecchi** [comment]
Yes, I had the same problem. There are some libraries that contain CCS values?
Thanks
